AdofaiTweaks/Tweaks/HideUiElements/HideUiElementsSettings.cs

```diff
@@ -86,16 +92,21 @@ public class HideUiElementsSettings : TweakSettings
                 AdofaiTweaks.ReleaseNumber >= 94 ? null : scnEditor.instance);
 
             if (isEditingLevel) {
-                if (scnEditor.instance?.ottoCanvas.gameObject.activeSelf == hideOtto) {
-                    scnEditor.instance.ottoCanvas.gameObject.SetActive(!hideOtto);
+                scnEditor.instance.autoImage.enabled = !hideOtto;
```

Comment:
I think this only hides the image, can you disable the button as well?

Reply:
Done, the button was indeed still working